pset-functions.js

```diff
@@ -175,6 +244,35 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+//Global variable for user inputted word
+const userInput = 'run'
+
+//Create variable and reverse user inputted word
+function reverseString(word) {
+   //Turn the word into individual letters in an array
+   const splitString = word.split('')
+   //Reverse the array elements
+   const reverseArray = splitString.reverse()
+   //Combine the array elements back into a word
+   const joinArray = reverseArray.join('')
+   return joinArray
+}
+
+//Global variable that holds reversed word
+const reversedString = reverseString(userInput)
+
+//Comparing original word to reversed word and returning True or False
+function isPalindrome(word) {
+   if(word === reversedString) {
+      return "True"
+   } else {
+      return "False"
+   }
+}
+
+const qualityCheck = isPalindrome(userInput)
+console.log(`Is your word input a palindrome? ${qualityCheck}.`)
+
```

Comment:
@cswebdesign Awesome Job with this solution. Good Idea separating out your logic into separate functions that you can reuse. One suggestion, with your global variable`reversedString` you can move them inside your isPalindrome function, replace the userInput parameter with `word`, and make it a local variable. Also in your return "True" and "False" statements you can remove the quotes and make it lowercase. this will turn the value from a string to a boolean.
Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L242)